docs/diagrams/FreeScheduleActivityDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
```

Comment:
I believe activity diagrams are meant to illustrate actions that the user take and also the high level responses of the system, rather than the internal workings of our product. Would it be better to remove this? Furthermore, this activity diagram seems to illustrate very similar information as the sequence diagram.

Reply:
I think the activity diagram would be fine here. The sequence diagram only describes one possible scenario that could have happened. However, since date could be optional, I think that it will be fine to include it here.

Reply:
The activity diagram is also an easier diagram to read to understand the flow of `freeSchedule` command

Reply:
Hmm, I am not very convinced, I believe parts like "User executes command" are on an entirely different layer of abstraction than parts like "Create isPersonFree predicate". Furthermore, the activity diagrams that we have been taught so far all show only the highest level actions, i.e. User takes some action, System responses as such etc. This diagram seems like a mix of high level and low level details.

Reply:
Yea makes sense, I will see if I can change it. Otherwise I will remove it